.gitignore

```diff
@@ -1 +0,0 @@
-.gitignore
```

Comment:
I only added -ar
.gitignore was deleted by mistake :\ , but I pulled the original one later.